Makefile

```diff
@@ -1,28 +1,40 @@
-GO = go
-pkgs = $(shell $(GO) list ./... | grep -v /vendor/)
-PROMU = promu
-BIN_DIR ?= $(shell pwd)
+GO           ?= GO15VENDOREXPERIMENT=1 go
```

Comment:
Is it for backwards compatibility?

Reply:
yep, we don`t need it anymore. See,
> Go 1.5 introduced experimental support for a "vendor" directory, enabled by the > GO15VENDOREXPERIMENT environment variable. Go 1.6 enabled this behavior by default, and in Go 1.7, this switch has been removed and the "vendor" behavior is > always enabled.
I will remove this tomorrow in another PR.